content/training/troubleshooting.mdx

```diff
@@ -52,6 +52,10 @@ There is a couple of possible reasons for such behavior:
 - When the answer is expected to be a floating-point number, it might not be precise enough to compare equal with the expected value, but assertion message rounds both so they look the same. Check for possible rounding errors or floating-point accuracy issues.
 - If all tests fail with some message similar to `"Expected 'Something' but got None"` or something along the lines (depending on language and assertions library used) it might mean that you only print your answer to the stdout instead of returning it from your function. This problem happens especially often for people who do not understand the difference between printing value and returning it, or when someone comes from some other competitive programming website, which actually requires values to be printed and not returned.
```

Comment:
Suggested fixes:

```suggestion
- When the answer is expected to be a floating-point number, it might not be precise enough to compare equal with the expected value, but the assertion message rounds both so they look the same. Check for possible rounding errors or floating-point accuracy issues.
- If all tests fail with some message similar to `"Expected 'Something' but got None"` or something along those lines (depending on language and assertions library used), it might mean that you only print your answer to the stdout instead of returning it from your function. This problem occurs especially often for people who do not understand the difference between printing a value and returning it, or when someone comes from some other competitive programming website, which actually requires values to be printed and not returned.
```